Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionTanFragment.kt

```diff
@@ -28,6 +29,16 @@ class SubmissionTanFragment : BaseFragment() {
         return binding.root
     }
 
+    override fun onResume() {
+        super.onResume()
+        activity?.window?.setFlags(WindowManager.LayoutParams.FLAG_SECURE, WindowManager.LayoutParams.FLAG_SECURE)
```

Comment:
Does it makes sense to do this based on flavors? e.g. we can still keep screen shots enabled for debug mode and disable them for release mode. This way communication with the design team can be easier while developing this app.

Reply:
I'd like to have screen shots for the non-privacy related stuff enabled, but disallow it for the TAN input and several other submission fragments.
I agree, in debug mode, screenshos should be enabled anywhere.

The main problem I have is the "hack" to enable or disable screenshots based on fragment. I will now open another issue to split the main activity (the app currently already has 2 activities: onboarding and "main") from the submission one. Then you can declare the "submission" activity's window as secured, while the main activity and the onboarding activity are not specifically secured. Of course a debug mode would really be good.

Reply:
I opened #77